cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -17,6 +17,27 @@
 # Copyright 2017 Red Hat, Inc.
 #
 
+wait_for_kubernetes() {
+    local timeout="$1"
```

Comment:
Maybe set a default in case it's missed i.e. ${1:-600}

Reply:
Timeout could be in the unit of minutes, then the loop below would be much simpler

Reply:
1. Will do.
2. I think that it will add more code to the loop because I'll have to divide  `start_time` and `current_time` by 60.
   what do you think?

Reply:
This doesn't need to be precise. I'm in favor of just looping 'x' number of 1 minute intervals.